tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -1344,7 +1344,9 @@ def output_size(self):
 
   def zero_state(self, batch_size, dtype):
     with ops.name_scope(type(self).__name__ + "ZeroState", values=[batch_size]):
-      return self._cell.zero_state(batch_size, dtype)
+      if (hasattr(self._cell, "zero_state")):
+        return self._cell.zero_state(batch_size, dtype)
+      return super(DropoutWrapper, self).zero_state(batch_size, dtype)
```

Comment:
I see. 

I think the proper fix should be in the get_initial_state() method in the wrapper, which should call self._cell.get_initial_state(batch_size=batch_size, dtype=dtype).